UefiCpuPkg/Universal/Acpi/S3Resume2Pei/S3Resume.c

```diff
@@ -1130,9 +1168,6 @@ S3RestoreConfig2 (
     if (((SmmS3ResumeState->Signature == SMM_S3_RESUME_SMM_32) && (sizeof (UINTN) == sizeof (UINT32))) ||
         ((SmmS3ResumeState->Signature == SMM_S3_RESUME_SMM_64) && (sizeof (UINTN) == sizeof (UINT64))))
     {
-      SmmS3ResumeState->MpService2Ppi = (EFI_PHYSICAL_ADDRESS)(UINTN)MpService2Ppi;

```

Comment:
Move below after locate the MpService2Ppi in this patch even MpService2Ppi is not used anymore in SMM CPU s3, we need another patch to clean the SmmS3ResumeState->MpService2Ppi.

SmmS3ResumeState->MpService2Ppi = (EFI_PHYSICAL_ADDRESS)(UINTN)MpService2Ppi;